modules/headline.py

```diff
@@ -1,28 +1,18 @@
-import urllib.request
-from bs4 import BeautifulSoup
 from textblob.classifiers import NaiveBayesClassifier
 from textblob import TextBlob
+import newspaper
 
 class title:
 
     #Initialisations
     def __init__(self): 
         self.news_url="https://edition.cnn.com/2019/08/25/politics/trump-g7-boris-johnson-emmanuel-macron/index.html"
 
-
     def extract_headline(self):
-        self.net_con=True #Expecting Internet Connection to be working initially
-        try:
-            news_page=urllib.request.urlopen(self.news_url)   
-            soup = BeautifulSoup(news_page,'html.parser')
-            headline_in_html=soup.find('h1')
-            headline=headline_in_html.text.strip()
-            return headline
-
-        except urllib.error.URLError:
```

Comment:
I am waiting for @mandjevant reply. I have requested for this block to be preserved. as it gives a page of error when there is no internet. Hence the variable "net_con" was required too

Reply:
The block wont work when Urllib isnt requesting a site. (Since we are using newspaper now)

Reply:
Additionally, you should make a seperate statement at the begining of the file that returns an error and sys.exit()'s the program.

Reply:
So I shouldn't make any changes now, right @mandjevant @asiffarhankhan ?

Reply:
@mphirke Regarding this, you should remove block and make a network connection check in __init__.